homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+@font-face {
+  font-family: "Actor-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Actor-Regular.woff2") format("woff2"), url("../fonts/Actor-Regular.woff") format("woff");
+}
+@font-face {
+  font-family: "Acme-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Acme-Regular.woff2") format("woff2"), url("../fonts/Acme-Regular.woff") format("woff");
+}
+@font-face {
+  font-family: "Aclonica-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Aclonica-Regular.woff2") format("woff2"), url("../fonts/Aclonica-Regular.woff") format("woff");
+}
+html {
+  box-sizing: border-box;
+}
+*, *::after, *::before {
+  box-sizing: inherit;
+}
+a {
+  text-decoration: none;
+  display: block;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
+}
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+button {
+  padding: 0;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: transparent;
+}
+body {
+  min-width: 375px;
+  overflow-x: hidden;
+  font-family: "Aclonica-Regular", sans-serif;
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+body.lock {
+  overflow: hidden;
+}
+.container {
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: 0 auto;
+  min-width: 375px;
+}
+.hidden {
+  display: none !important;
+}
+.logo__link {
+  font-family: "Actor-Regular";
+  color: #fff;
+  font-size: 26px;
+  line-height: 26px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-weight: bold;
+  letter-spacing: 1.2px;
+}
+.header {
+  z-index: 1;
+  padding: 18px 20px;
+}
+.header .menu__list {
+  display: flex;
+  flex-direction: column;
+}
+.header .menu__list > li {
+  position: relative;
+  margin: 0 auto;
+}
+.header .menu__link {
+  font-size: 12px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+}
+.header .header__button-link {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  top: 80px;
+}
+.header .header__button-link a {
+  font-size: 12px;
+  border-radius: 20px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+  padding: 8px 0;
+  border: 1.5px solid #22BF95;
+  border-radius: 20px;
+}
+.header .header__banner {
+  height: 211px;
```

Comment:
Please, avoid hardcoded sizes. It is good practice to set height with paddings. 
In your case you can remove paddings from .header__banner-title and set to your .header__banner. 
Try to keep this approach in mind and apply it in future.

Reply:
done